libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -74,39 +74,39 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     // @Units: m
     // @Range: 1 1000
     // @User: Standard
-    AP_GROUPINFO("_DIST_MAX", 5, AP_Follow, _dist_max, 100),
+    AP_GROUPINFO("_DIST_MAX", 5, AP_Follow, _dist_max_m, 100),
 
     // @Param: _OFS_TYPE
     // @DisplayName: Follow offset type
     // @Description: Follow offset type
-    // @Values: 0:North-East-Down, 1:Relative to lead vehicle heading
+    // @Values: 0:North-East-Down, 1:Relative to lead vehicle heading_deg
     // @User: Standard
     AP_GROUPINFO("_OFS_TYPE", 6, AP_Follow, _offset_type, AP_FOLLOW_OFFSET_TYPE_NED),
 
     // @Param: _OFS_X
-    // @DisplayName: Follow offsets in meters north/forward
-    // @Description: Follow offsets in meters north/forward.  If positive, this vehicle fly ahead or north of lead vehicle.  Depends on FOLL_OFS_TYPE
+    // @DisplayName: Follow offsets_ned_m in meters north/forward
+    // @Description: Follow offsets_ned_m in meters north/forward.  If positive, this vehicle fly ahead or north of lead vehicle.  Depends on FOLL_OFS_TYPE
```

Comment:
This makes the user facing docs harder to read IMHO.

Reply:
I agree, this was accidental.